client/my-sites/invites/invite-accept-logged-in/index.jsx

```diff
@@ -86,11 +86,11 @@ class InviteAcceptLoggedIn extends Component {
 	renderMatchEmailError = () => {
 		return (
 			<Card>
-				<InviteFormHeader { ...this.props.invite } user={ this.props.user } matchEmailError />
```

Comment:
As far as I can tell, this prop doesn't do anything anymore, which is why I removed it.